docs/services/data/dataset.md

```diff
@@ -41,26 +41,7 @@ To interact with datasets programmatically, use the [data client API](/appendix/
 
 {{< readfile "/static/include/app/apis/generated/dataset-table.md" >}}
 
-## Sample dataset
-
-If you are testing, you can use one of two provided sample dataset, one of shapes and the other of a wooden figure:
-
-{{<imgproc src="/tutorials/data-management/shapes-dataset.png" resize="1200x" declaredimensions=true style="max-width:400px" alt="The shapes dataset." class="imgzoom fill aligncenter">}}
-
-{{< imgproc src="/tutorials/filtered-camera-module/viam-figure-dataset.png" style="max-width:400px" alt="The datasets subtab of the data tab in the Viam app, showing a custom 'viam-figure' dataset of 25 images, most containing the wooden Viam figure" class="imgzoom fill aligncenter" resize="1400x" >}}
-
-1. [Download the shapes dataset](https://storage.googleapis.com/docs-blog/dataset-shapes.zip) or [download the wooden figure dataset](https://storage.googleapis.com/docs-blog/dataset-figure.zip).
-1. Unzip the download.
-1. Open a terminal and go to the dataset folder.
-1. In it you will find a Python script to upload the data to the Viam app.
-1. Open the script and fill in the constants at the top of the file.
-1. Run the script to upload the data into a dataset in Viam app:
-
-   ```sh {class="command-line" data-prompt="$" }
-   python3 upload_data.py
-   ```
-
-## Next steps
+## How-to guide
```

Comment:
It just occurs to me that we should also probably link to the CLI https://docs-test.viam.dev/3473/cli/#dataset . Maybe under the API header we should add a CLI header with an example and a link to the dataset section?

Reply:
Step 3 in the how-to does link to the CLI command for creating a dataset but I agree I will add mention in the intro of this page as well as a CLI header. IDK about an example though...the dataset section of CLI has tons of examples of dataset command usage

Reply:
Now I'm thinking CLI header doesn't make sense on the page because there wouldn't be much content in it, so I'm just adding a link to the command in the intro of this page.

Reply:
But now the heading makes it sound like you cannot manage datasets with the API...

Reply:
I don't follow. Which heading? I am now adding something to the intro paragraph of this page saying you can use the data client API

Reply:
sorry - I meant to say the intro paragraph